app/web/features/profile/view/ReferencesView.tsx

```diff
@@ -65,14 +67,14 @@ export default function ReferencesView({
                 loading={isFetchingNextPage}
                 onClick={() => fetchNextPage()}
               >
-                {SEE_MORE_REFERENCES}
+                {t("profile:see_more_references")}
```

Comment:
I'd probably consider renaming this since this is named quite literal to the button text as is right now. (e.g. what if we re-word this to "Show more references"?) This is a bad example of how we should name things from our older code/more naive self... 😅